scipy/optimize/_nonlin.py

```diff
@@ -675,7 +679,7 @@ def append(self, c, d):
         if len(self.cs) > c.size:
             self.collapse()
 
-    def __array__(self):
+    def __array__(self, copy=None):
```

Comment:
`dtype=None` is used in every other signature change in the PR but not here - is that deliberate? Otherwise LGTM

Reply:
Good catch. Tests didn't fail and my code search skills apparently fell short.

This code doesn't really seem to be used, I'll add the keyword with a warning that any input that's not `None` is not handled.